docs/Import-Gene-Panels.md

```diff
@@ -6,15 +6,15 @@ This page describes how to import a gene panel into the cBioPortal database.  It
 2. The gene panel to import is in the proper file format.  See [File Format](File-Formats.md#gene-panel-data) for more information.
 3. The `PORTAL_HOME` environment variable has been properly defined.  See [Pre-Build-Steps](Pre-Build-Steps.md#set-the-portal-home-variable) for more information.
 
-The following command 2 commands can be used to import a gene panel into the cBioPortal database. In this example, we are loading the example gene panel which resides in the sample dataset `study_es_0`.
+The following 2 methods can be used to import a gene panel into the cBioPortal database. In this example, we are loading the example gene panel which resides in the sample dataset `study_es_0`.
```

Comment:
please call it command. Method 1 vs method 2 makes me think there are 2 alternatives for doing this